src/assets/scripts/client/scope/ScopeModel.js

```diff
@@ -0,0 +1,208 @@
+import _has from 'lodash/has';
+import _isNil from 'lodash/isNil';
+import EventBus from '../lib/EventBus';
+import NavigationLibrary from '../navigationLibrary/NavigationLibrary';
+import RadarTargetCollection from './RadarTargetCollection';
+import { EVENT } from '../constants/eventNames';
+import { DECIMAL_RADIX } from '../constants/globalConstants';
+import { THEME } from '../constants/themes';
+
+export default class ScopeModel {
+    constructor() {
+        this._eventBus = EventBus;
+        this._navigationLibrary = NavigationLibrary;
+        // TODO: Use this!
+        this._sectorCollection = [];
+        this._theme = THEME.DEFAULT;
+
+        this.radarTargetCollection = [];
+
+        this._init();
+    }
+
+    /**
+     * Complete initialization tasks
+     *
+     * @for ScopeModel
+     * @method _init
+     */
+    _init() {
+        this.radarTargetCollection = new RadarTargetCollection(this._theme);
+    }
+
+    /**
+     * Accept a pending handoff from another sector
+     *
+     * @for ScopeModel
+     * @method acceptHandoff
+     * @param radarTargetModel {RadarTargetModel}
+     * @return result {array} [success of operation, system's response]
+     */
+    acceptHandoff(radarTargetModel) {
+        return [false, 'acceptHandoff command not yet available'];
+    }
+
+    /**
+     * Amend the cruise altitude OR interim altitude for a given `RadarTargetModel`
+     *
+     * @for ScopeModel
+     * @method amendAltitude
+     * @param radarTargetModel {RadarTargetModel}
+     * @param altitude {string}
+     * @return result {array} [success of operation, system's response]
+     */
+    amendAltitude(radarTargetModel, altitude) {
+        altitude = parseInt(altitude, DECIMAL_RADIX);
+
+        return radarTargetModel.amendAltitude(altitude);
+    }
+
+    /**
+     * Enable handlers
+     *
+     * @for ScopeModel
+     * @method disable
+     */
+    disable() {
+        this._eventBus.off(EVENT.SET_THEME, this._setTheme);
+    }
+
+    /**
+     * Disable handlers
+     *
+     * @for ScopeModel
+     * @method enable
+     */
+    enable() {
```

Comment:
move to after `_init()` with other lifecycle methods defined in calling order.  does this method ever get called?

Reply:
done, and fixed